tutorials/automate-tasks-using-cron/index.mdx

```diff
@@ -0,0 +1,186 @@
+
+| MINUTE (0 - 59) 	| HOUR (0 - 23, 0 = MIDNIGHT) 	| DAY OF THE MONTH (1 - 31) 	| MONTH (1 - 12) 	| DAY OF THE WEEK (0 - 6, 0 = SUNDAY) 	| COMMAND OR DIRECTORY PATH AND SCRIPT NAME 	|
+|-----------------	|-----------------------------	|---------------------------	|----------------	|-------------------------------------	|-------------------------------------------	|
+| *                	| *                            	| *             	            | *               	| *                                   	| e.g. /var/www/websites/backup.sh          	|
+
+<Message type="tip">
+
+You can also use the following notation for:
+* Months: `JAN-DEC`
+* Day of the week: `SUN-SAT`
+
+</Message>
+
+Following an example that runs a backup every night at 4:30: 
+
+```
+30 04 * * * /var/www/websites/backup.sh
+```
+
+### Special strings
+
+Cron provides a range special strings, which can be used in place of the five time-and-date fields:
+
+| String       	| Meaning                                                                                                          	|
+|--------------	|------------------------------------------------------------------------------------------------------------------	|
+| `*`          	| Wildcard variable that represents “all”, *` * * * *` run every minute of every hour of every day of every month. 	|
+| `,`          	| Break up scheduling values to form a list, `0,30 * * * *` run at the beginning and the middle of every hour.     	|
+| `-`          	| Range of values in the schedule field, `0-29 * * * *`  run very minute of the first 30 min of every hour.        	|
+| `*/`         	| Express a step value,`*/10 * * * *` run on every minute divisible by 10 (10, 20, 30 etc.).                       	|
+| `@reboot`   	| Run once, at startup.                                                                                            	|
+| `@yearly`   	| Run once a year, `0 0 1 1 *`.                                                                                    	|
+| `@annually` 	| (same as `@yearly`)                                                                                              	|
+| `@monthly`  	| Run once a month, `0 0 1 * *`.                                                                                   	|
+| `@weekly`   	| Run once a week, `0 0 * * 0`.                                                                                    	|
+| `@daily`    	| Run once a day, `0 0 * * *`.                                                                                     	|
+| `@midnight` 	| (same as `@daily`)                                                                                               	|
+| `@hourly`   	| Run once an hour, `0 * * * *`.                                                                                   	|
+
+## Managing crontabs
+
+Once you have decided when you want to run your cronjobs, privde the info in your `crontab` file so that the daemon can read it. 
```

Comment:
```suggestion
Once you have decided when you want to run your cronjobs, provide the info in your `crontab` file so that the daemon can read it. 
```